Python/compile.c

```diff
@@ -5359,9 +5359,10 @@ compiler_subscript(struct compiler *c, expr_ty e)
         case AugStore:/* fall through to Store */
         case Store:   op = STORE_SUBSCR; break;
         case Del:     op = DELETE_SUBSCR; break;
-        case Param:
-            PyErr_SetString(PyExc_SystemError,
-                "param invalid in subscript expression");
+        default:
+            PyErr_Format(PyExc_SystemError,
+                         "expr_context kind %d should not be possible",
```

Comment:
I find the error message a bit odd. I would suggest changing it to something like:

"Received unexpected expr_context kind: %d"

Reply:
This message is for consistency with other usages in the compiler. An example is mod_ty switch in compiler_mod